smallest_multiple.js

```diff
@@ -6,9 +7,18 @@
  * @return { Number }         Lowest Positive Number that is evenly divisible by all numbers
  *                            between 1 and `ceiling`
  */
-module.exports = function( ceiling ) {
-  // do work here
-
-
-  return 0;
-};
\ No newline at end of file
+ module.exports = function( ceiling ) {
+    for(var i = 1; i <= 10000000000000; i++){
+      let passes = true;
+      for(var j = 1; j <= ceiling; j++){
+        if (i % j !== 0) {
+          passes = false;
+            break;
+          }
+        }
+        if (passes === true) {
+          return i;
+        }
+      }
+      return i;
```

Comment:
Referencing a value i that is outside the block for the for loop in which it was declared.